### ISSUE TITLE
new front page rotation

### ISSUE THREAD

**Issue by ValWood** [pombase/website#923]
Caption
The Secret Behind Cell Revival Revealed

Image, in folder yangida. File name: Yanagida_mitotic_restart

paper
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6093628/

**Comment by kimrutherford** [comment]
There are no annotations for that paper in Chado.  Should I wait until tomorrow before adding the Spotlight item?

**Comment by ValWood** [comment]
yes sorry, I just approved, to for tomorrow

**Comment by kimrutherford** [comment]
I've just checked in the new Spotlight item so the annotations and spotlight should be live at the same time.

**Comment by kimrutherford** [comment]
It's live now.

**Comment by ValWood** [comment]
can I be a pain. I added quite a lot of annotation here, and I can't put it through because I'm waiting on a few terms. This means the publication page link won't work. Could we remove the rotation for a few days. When it comes back we can also tweet it.

**Comment by kimrutherford** [comment]
> This means the publication page link won't work.

The link works:
https://www.pombase.org/reference/PMID:30116786

> Could we remove the rotation for a few days.

I'll do that now.  It will be updated in 10-15 minutes.

**Comment by ValWood** [comment]
Yep it works today, I put the session through before I made my additions. 
I did not think there would be many, but there were quite a few...

**Comment by ValWood** [comment]
I'll  let you know when I finalise the session and we'll reinstate

**Comment by ValWood** [comment]
OK this one is now re-approved with the  added annotations. Front page rotation can go live tomorrow. Then we can retweet


https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.oist.jp%2Fnode%2F33027&text=The%20Secret%20Behind%20Cell%20Revival%20Revealed

with "now curated in PomBase" with paper link etc

**Comment by kimrutherford** [comment]
I've checked in the new item.  I'll check it tomorrow.
